### ISSUE TITLE
stage damage functions are not matching b/w 1.4.3 and 2.0

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#701]
Stage-damage functions with uncertainty are not matching between the two versions. Take a look at the residential stage-damage function with uncertainty computed using 1.4.3 below. The mean damage for content plus structure at a stage of 947 feet is about $61k. 

![image](https://user-images.githubusercontent.com/25163165/206927451-82121044-9628-4156-97b1-c5bbc19fff7d.png)

While mean damage for structure alone at a stage of 947 feet is $112k according to the stage-damage function with uncertainty computed using 2.0. I think the jaggedness is because of a very small number of iterations. 

![image](https://user-images.githubusercontent.com/25163165/206927529-2f7b7eaa-4a2d-4243-86e6-8d87ff0f6429.png)

I think we can say with a relatively high level of certainty that this is related to an unknown difference between the implementation of uncertainty in 1.4.3 and that which has been implemented in 2.0. We have shown that a deterministic stage-damage compute - including aggregation - has been implemented correctly. We have also show that damage with uncertainty at a given coordinate has been implemented correctly, as have the constituent uncertainty-related calculations. What's more, the stage-damage structure details are identical: 

![image](https://user-images.githubusercontent.com/25163165/206927840-2efa2b62-ab89-45bd-939f-c3ba623ceaf6.png)

In summary: 
- In 2.0 we calculate damage the same way as in 1.4.3
- We aggregate damage correctly in 2.0
- We sample uncertain parameters correctly in 2.0
- Stage-damage functions with uncertainty do not match
